tests/api/test_dynamo.py

```diff
@@ -0,0 +1,227 @@
+from hyp3_api import dynamo
+
+
+def test_query_jobs_by_user(tables):
+    table_items = [
+        {
+            'job_id': 'job1',
+            'user_id': 'user1',
+            'job_type': 'fake job type',
+            'status_code': 'fake job stats',
+            'request_time': '2000-10-01T00:00:00+00:00'
+        },
+        {
+            'job_id': 'job2',
+            'user_id': 'user1',
+            'job_type': 'fake job type',
+            'status_code': 'fake job stats',
+            'request_time': '2000-01-01T00:00:00+00:00'
+        },
+        {
+            'job_id': 'job3',
+            'user_id': 'user2',
+            'job_type': 'fake job type',
+            'status_code': 'fake job stats',
+            'request_time': '2000-01-01T00:00:00+00:00'
+        },
+    ]
+    for item in table_items:
+        tables['jobs_table'].put_item(Item=item)
+
+    response = dynamo.query_jobs('user1')
+    assert len(response) == 2
+    assert [i for i in response if i not in table_items[:2]] == []
```

Comment:
is this just `assert response == table_items[0:2]`, e.g. we got the first and second items back in our list?

Reply:
or even `assert dynamo.query_jobs('user1') == table_items[0:2]`